BUILD.md

```diff
@@ -1,163 +1,162 @@
 
-If you have all above - run the tests (both unit and integration):
+If you have all above - build and run the tests (both unit and integration):
 ```
-heroku restart --app ct-payone-integration-test # may be skipped on secondary run
-sbt clean test it:test
+heroku restart --app ct-payment-integration-java # may be skipped on secondary run
+./gradlew clean build
 ```
 
-The tests some times fail because of this [known issue](#known-issues).
+To run all these tests locally (including IDE) with mandatory properties above - use `it.properties` file 
+(in `it/resources` directory) with all secret values. See `ItConfig.java` for more details. 
+Also you could copy and edit `it/resources/it.properties.skeleton`.
 
-# Publish workflow
+The gradle _build_ task is configured to be depended on all the tests, including Unit tests on all sub-projects and 
+common integration tests from [`src/`](/src/) directory.
 
-## Publish to local maven repo
- 
-This step may be used local tests of SNAPSHOT version:
-```
-sbt clean test it:test publishM2
-```
+The tests some times fail because of this [known issues](#known-issues).
 
-## Publish to Maven
-
-### **Note**: set proper version in `version.sbt` before publishing to central repo.
-
-If you are a new developer in the project - update contributors list in `build.sbt -> commonSettings -> pomExtra`.
+# Publish workflow
 
-The target repo is defined by _nexusHost_ variable in [build.sbt](build.sbt)
+## Full build with tests, documentation publishing and Bintray upload
 
-If you want to publish just test internal (SNAPSHOT) version - better to publish to 
-http://repo.ci.cloud.commercetools.de/ first ([see the instructions below](#5-publish-to-repo.ci.cloud.commercetools.de)),
-and then if the version is ready to be public released - publish to Maven Central.
+Replace `X.X.X` in the snippet:
 
-Publishing to Maven Central requires the next steps:
+```
+./gradlew clean build aggregateJavaDoc gitPublishPush bintrayUpload -Dbuild.version=X.X.X
+```
 
- 1. Build the app (see the steps above for integration tests)
- 2. [Signing up the app with PGP key](#2-signing-up-the-app-with-pgp-key)
- 3. [Deploy to OSS Sonatype](#3-deploy-to-oss-sonatype)
- 4. [Manually release from Sonatype web page to Maven Central](#4-manually-release-from-sonatype-web-page-to-maven-central)
- 
-**Note**: to publish only to Sonatype (both OSS or commercetools) PGP signing is not required.
+## Publish to local maven repo
  
-### 2. Signing up the app with PGP key
-
-This step requires installed [PGP tool](https://gpgtools.org/), 
-[published PGP key](http://security.stackexchange.com/questions/406/how-should-i-distribute-my-public-key) and 
-[configured SBT PGP plugin](http://www.scala-sbt.org/sbt-pgp/). 
-
-More info about SBT publishing find here:
-
-http://www.scala-sbt.org/0.13/docs/Using-Sonatype.html
+This step may be used for local test versions:
+```
+./gradlew clean build publishPaymentPublicationPublicationToMavenLocal
```

Comment:
actually, yes - this is correct task name, but now changed to `install` (it works with maven gradle plugin)